kyo-core/shared/src/main/scala/kyo/typeMap.scala

```diff
@@ -3,14 +3,14 @@ package kyo
 import kyo.Tag.Intersection
 import scala.collection.immutable.HashMap
 
-opaque type TypeMap[+A] = HashMap[Tag[?], Any]
+opaque type TypeMap[+A] = HashMap[Tag[Any], Any]
 extension [A](self: TypeMap[A])
 
-    private inline def fatal(using t: Tag[?]): Nothing =
+    private inline def fatal[T](using t: Tag[T]): Nothing =
         throw new RuntimeException(s"fatal: kyo.TypeMap of contents [$self] missing value of type: [${t.show}].")
 
     def get[B >: A](using t: Tag[B]): B =
-        val b = self.getOrElse(t, null)
+        val b = self.getOrElse(t.erased, null)
```

Comment:
`erased` transforms a `Tag[T]` into `Tag[Any]`

Reply:
We can probably make this function accept `Tag[Any]`, right? The `.show` should still be accurate.